#15/clear message history

gagpa opened the pull request.

gagpa committed:
Фикс отчистки истории сообщений пользователя по команде /hide

gagpa committed:
Фикс отчистки истории сообщений пользователя по расписанию

- Переименован класс для удаления сообщений
- Улучшено оформление кода

gagpa committed:
Удалены не используемые классы

spamelick commented on app/scheduler/__init__.py:
```diff
@@ -1,11 +1,15 @@
-                                 expire_message_time=configs.MESSAGE_EXPIRE_TIME,
-                                 )
+async def run():
+    """Запускает расписание"""
```
Переформулировать: Запуск расписания

spamelick commented on app/modules/user_module/message_history_cleaner.py:
```diff
@@ -0,0 +1,41 @@
+
+
+class UserHistoryCleaner:
+    """Отчистка истории сообщений пользователей"""
```
Очистка

gagpa committed:
Фикс выводимого сообщения после команды /start

- Исправлена орфографическая ошибка в слове очистка
- Исправлен док стринг в функцие для запуска расписания
